adele-booking/src/main/java/com/microservicesteam/adele/booking/infrastucture/config/GuavaModuleConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.microservicesteam.adele.booking.infrastucture.config;
+
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class GuavaModuleConfig {
+
+    @Bean
+    public Module guavaModule() {
```

Comment:
@bearcheese I think it is good enough. On SPI we registered it the way below:
`ObjectMapper mapper = new ObjectMapper();
mapper.registerModule(new GuavaModule());`

Your solution is more elegant. 👍